CTC/src/ctc/model/Route.java

```diff
@@ -0,0 +1,29 @@
+package ctc.model;
+
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.ListIterator;
+
+import trackmodel.model.Block;
+
+public class Route {
+
+  private LinkedList<Block> path;
+  private Block current;
```

Comment:
@michaelplazek This is never used. What do you plan on using it for? If not much, why not just use a linked list?